### DISCUSSION TITLE
ant-steps-item-icon color question.

### DISCUSSION THREAD

**Discussion by pipechaves** [ant-design/ant-design#47381] [Q&A] [answered]
Im using my css as a module, so YOURCSSNAME.module.css on typescript.

I wan to change the color of the step icon with a custom one

so far Im unable to do so , the only way to do it is by using the :global on the css module.

I want to achive it using the className of the icon.

```
import React from 'react';
import styles from './Steps.module.css';
import { Steps } from 'antd';

const { Step } = Steps;

// Example usage
const MyComponent: React.FC = () => {
  const customDot = <div className={styles.customstepsdot}></div>;
  return (
    <>
    <Steps type="inline">
      <Step title="Step 1" className='customstepsdot'/>
      <Step title="Step 2"   icon={customDot} />
      <Step title="Step 3"   icon={customDot} />
    </Steps>
    </>
  );
};
```
In order for it to be like
        
![image](https://github.com/ant-design/ant-design/assets/22892247/8847dd77-be8e-4a1b-a477-1497f9842d86)

and not the default blue color.
![image](https://github.com/ant-design/ant-design/assets/22892247/89c2fddc-42ec-4b18-9a95-3df15b0a3029)

I'm not using theme or nothing like that on the project.

if some one can guide me on it.

regards.

**Comment by Wxh16144** [comment, marked as answer]
> I want to achive it using the className of the icon.

As far as I know, this is not supported.

If you're using a css module, the only way around this is to use `:global`.

**Reply by pipechaves** [reply]
thanks
